ambari-server/src/main/python/ambari_server/setupSecurity.py

```diff
@@ -1127,3 +1137,61 @@ def migrate_ldap_pam(args):
   else:
     print_info_msg('LDAP to PAM migration completed')
   return retcode
+
+
+def populate_ambari_requires_ldap(options, properties):
+  if options.ldap_enabled_ambari is None:
+    enabled = get_boolean_from_dictionary(properties, AMBARI_LDAP_AUTH_ENABLED, False)
+    enabled = get_YN_input("Use LDAP authentication for Ambari [y/n] ({0})? ".format('y' if enabled else 'n'), enabled)
+  else:
+    enabled = 'true' == options.ldap_enabled_ambari
+
+  properties[AMBARI_LDAP_AUTH_ENABLED] = 'true' if enabled else 'false'
+
+
+def populate_service_management(options, properties, ambari_properties, admin_login, admin_password):
+  services = ""
+  if options.ldap_enabled_services is None:
+    if options.ldap_manage_services is None:
+      manage_services = get_boolean_from_dictionary(properties, LDAP_MANAGE_SERVICES, False)
+      manage_services = get_YN_input("Manage LDAP configurations for eligible services [y/n] ({0})? ".format('y' if manage_services else 'n'), manage_services)
+    else:
+      manage_services = 'true' == options.ldap_manage_services
+      stored_manage_services = get_boolean_from_dictionary(properties, LDAP_MANAGE_SERVICES, False)
+      print("Manage LDAP configurations for eligible services [y/n] ({0})? {1}".format('y' if stored_manage_services else 'n', 'y' if manage_services else 'n'))
+
+    if manage_services:
+      enabled_services = get_value_from_dictionary(properties, LDAP_ENABLED_SERVICES, "").upper().split(',')
+
+      all = "*" in enabled_services
```

Comment:
Can `WILDCARD_FOR_ALL_SERVICES ` be used here?

Reply:
Fixed; thanks!